sql/src/test/java/io/crate/integrationtests/Setup.java

```diff
@@ -268,6 +268,14 @@ public void createTestTableWithPrimaryKey() {
         transportExecutor.ensureGreen();
     }
 
+    public void createTestTableWithDefaultExpressions() {
```

Comment:
Given that this is only used once I think it would be better to inline it, so that it is easier visible in the test how the table looks like. 

The `ensureGreen` is also not required.